libs/ui-watt/src/lib/components/checkbox/watt-checkbox.component.ts

```diff
@@ -14,103 +14,68 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-import { Component, forwardRef, ViewEncapsulation } from '@angular/core';
 import {
-  ControlValueAccessor,
-  UntypedFormControl,
-  NG_VALUE_ACCESSOR,
-  ReactiveFormsModule,
-} from '@angular/forms';
-import {
-  MatLegacyCheckboxChange as MatCheckboxChange,
-  MatLegacyCheckboxModule as MatCheckboxModule,
-} from '@angular/material/legacy-checkbox';
-
-const customValueAccessor = {
-  multi: true,
-  provide: NG_VALUE_ACCESSOR,
-  useExisting: forwardRef(() => WattCheckboxComponent),
-};
-
-const selector = 'watt-checkbox';
+  Component,
+  ElementRef,
+  HostBinding,
+  ViewEncapsulation,
+  forwardRef,
+  inject,
+} from '@angular/core';
+import { ControlValueAccessor, NG_VALUE_ACCESSOR, FormsModule } from '@angular/forms';
 
 @Component({
+  selector: 'watt-checkbox',
+  styleUrls: ['./watt-checkbox.component.scss'],
+  template: `<label>
+    <input
+      [(ngModel)]="model"
+      [disabled]="isDisabled"
+      (ngModelChange)="onChange($event)"
+      type="checkbox"
+    />
+    <ng-content />
+  </label>`,
+  standalone: true,
   encapsulation: ViewEncapsulation.None,
-  selector,
-  styles: [
-    `
-      ${selector} .mat-checkbox-frame {
-        border-color: var(--watt-color-primary);
-      }
-
-      ${selector} {
-        line-height: 16px;
-      }
-    `,
+  providers: [
+    {
+      provide: NG_VALUE_ACCESSOR,
+      useExisting: forwardRef(() => WattCheckboxComponent),
+      multi: true,
+    },
   ],
-  templateUrl: './watt-checkbox.component.html',
-  providers: [customValueAccessor],
-  standalone: true,
-  imports: [MatCheckboxModule, ReactiveFormsModule],
+  imports: [FormsModule],
 })
 export class WattCheckboxComponent implements ControlValueAccessor {
```

Comment:
issue: validation is missing. Should that be part of this PR?

From Figma

![image](https://github.com/Energinet-DataHub/greenforce-frontend/assets/1096332/265ff6d1-5243-4e0a-8aa7-3a79026fd1f2)

Reply:
I´m missing that :-)